draft-ietf-mls-protocol.md

```diff
@@ -3634,8 +3638,8 @@ message at the same time, by taking the following steps:
   parameters:
   * `psktype`: `resumption`
   * `usage`: `reinit`
-  * `group_id`: The group ID for the current group
-  * `epoch`: The epoch that the group will be in after this Commit
+  * `group_id`: The group ID for the new group
+  * `epoch`: The epoch that the group will be in after this Commit (0x0)
```

Comment:
```suggestion
  * `group_id`: The group ID for the current group
  * `epoch`: The epoch that the group will be in after this Commit
```

Reverting because the PSK here refers to the last epoch of the earlier group.  So it has to use the old group's information.